### PR TITLE
Make use of the default config when testing.

### PR DESCRIPTION
**Pull request by forslund**
## Description
Always use the default config as base config to make tests less error prone when changes occur.

## How to test
Make sure Travis tests still works.

## Contributor license agreement signed?
CLA [Yes]

### CODE REVIEW

File: test/util.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+from mycroft.configuration.config import LocalConf, DEFAULT_CONFIG
+from copy import copy
+
+__config = LocalConf(DEFAULT_CONFIG)
+
+
+# Base config to use when mocking
+def base_config():
+    return copy(__config)
```
Comment (MatthewScholefield) [inline test/util.py]: Should this be `deepcopy` so that merged attribute of sub elements don't mutate the original?

Comment (forslund) [reply test/util.py]: If you think so it probably should, my python knowledge breaks down around here :) Will fix

Comment (MatthewScholefield) [reply test/util.py]: Only difference is:
```python
a = {'1': {'2': 'original'}}
b = copy(a)
c = deepcopy(a)

a['1']['2'] = 'altered'

print(b['1']['2'])  # altered
print(c['1']['2'])  # original
```

Comment (forslund) [reply test/util.py]: Thanks.

**Pull request merged**